packages/services/server/src/graphql-handler.ts

```diff
@@ -268,37 +268,19 @@ export const graphqlHandler = (options: GraphQLHandlerOptions): RouteHandlerMeth
   return async (req, reply) => {
     const requestIdHeader = req.headers['x-request-id'] ?? reqIdGenerate();
     const requestId = cleanRequestId(requestIdHeader);
-    let controller = abortControllerCache.get(req.socket);
```

Comment:
no abort?

Reply:
we dont need to add our own abortController, we  already have the abortcontroller from whatwg-node/server on the context request object.

Reply:
I changed the code for the mutex to use that one instead